src/utils/withWidth.js

```diff
@@ -13,24 +13,24 @@ import { keys } from '../styles/breakpoints';
  * @param breakpoint
  * @param inclusive - defaults to true
  */
-export const isWidthUp = (screenWidth, breakpoint, inclusive = true) => {
+export const isWidthUp = (breakpoint, screenWidth, inclusive = true) => {
```

Comment:
It is an arguments rename (only) that I did earlier for clarity and misunderstood. This is just making it right, the root cause of the issue, and I did check the usage in tabs. The signature is actually the same.